en/core-api/bank-transfer.md

```diff
@@ -1,69 +1,62 @@
 # Core API Bank Transfer Integration
 <hr>
-Basic integration process of Bank Transfer (Virtual Account) will be explained below.
-
-?> Please make sure you have already done [creating your Midtrans Account](/en/midtrans_account/overview), before proceeding with this section.
-
-One of the payment method offered by Midtrans is Bank Transfer. By using this payment method, customers will have the option to make a payment via bank transfer and Midtrans will send real time notification when the customer complete the payment.
-
-At this moment, Midtrans has integrated with the following bank transfer payment methods:
-
-1. BCA Virtual Account
-2. BNI Virtual Account
-3. BRI Virtual Account
-4. Mandiri Bill Payment
-5. Permata Virtual Account
-
-## Integration Step
-1. Send transaction data to API Charge.
-2. Display virtual account number and expiry time.
-3. Handling Post-Transaction.
-
+Basic integration process for Bank Transfer (Virtual Account) is explained in this section.
+Your customers can make payments using the <i>Bank Transfer</i> payment method provided by Midtrans. You will be notified when customer completes the transaction using this option. A list of banks supported by Midtrans is given below.
```

Comment:
> A list of banks supported by Midtrans is given below

Please add  this following details:

> A list of banks supported acquiring bank by Midtrans is given below. Although, it can accept transfer from any other banks, as long as it is real time transfer.